eslint-rules/lib/rules/assets-deprecation.js

```diff
@@ -0,0 +1,127 @@
+const _ = require('lodash');
+const utils = require('../utils');
+
+const MAP_SCHEMA = {
+  type: 'object',
+  additionalProperties: true,
+};
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'asset is deprecated',
+      category: 'Best Practices',
+      recommended: true,
+    },
+    messages: {
+      uiLib: 'This asset is deprecated.',
+    },
+    fixable: 'code',
+    schema: [MAP_SCHEMA],
+  },
+  create(context) {
+    function reportDeprecatedAssets(node, options) {
+      try {
+        const {dueDate} = context.options[0];
+        const dueDateNotice = dueDate ? ` Please fix this issue by ${dueDate}!` : '';
+        const msg = `'${options.path}' is deprecated. ${options.message}${dueDateNotice}`;
+        context.report({
+          node,
+          message: `${msg}`,
+          fix(fixer) {
+            if (options.fix) {
+              // const fix = Object.values(options.fix);
+              // fixer.replaceText(node, `${fix}`);
+              // fixer.replaceTextRange(node.range, fix);
+              // fixer.remove(node.object);
+            }
+          },
+        });
+      } catch (err) {
+        console.log('Found error in: ', context.getFilename());
+      }
+    }
+
+    // Path
+    let pathString = '';
+
+    function deprecationCheck(node) {
+      if (node) {
+        const check = isAssetsObject(node);
+        if (check && pathString !== '') {
+          const deprecatedObject = getDeprecatedObject(pathString);
+          if (deprecatedObject) {
+            const path = deprecatedObject.path;
```

Comment:
you can destruct the values here
`const {path, message, fix}  = deprecatedObject`